Projects/UOContent/Items/Lights/Lantern.cs

```diff
@@ -6,10 +6,12 @@ namespace Server.Items;
 [SerializationGenerator(0, false)]
 public partial class Lantern : BaseEquipableLight
 {
+    private static TimeSpan FullDuration = TimeSpan.FromMinutes(20);
```

Comment:
```suggestion
    protected static TimeSpan FullDuration = TimeSpan.FromMinutes(20);
```

Reply:
I think this change has broken the build. `FullDuration` is used by `OilFlask` when refueling.